public/scripts/openai.js

```diff
@@ -3188,6 +3227,13 @@ async function onModelChange() {
     eventSource.emit(event_types.CHATCOMPLETION_MODEL_CHANGED, value);
 }
 
+function getOpenRouterPromptFormats() {
+    return new Map([
```

Comment:
Sounds good, will give it a look.

Reply:
The library is quite nice, but I haven't found an application as I always require an exact match. Or did you have something specific in mind?

Reply:
This is possible, but it could clash with custom presets that are named similarly enough. On the other hand, this should not be a big problem as people who create their own presets will probably disable the option.